From 84d03f57a651f5f0014c3afd28d212fa2a760861 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Wed, 25 Apr 2018 16:34:13 +0200
Subject: [PATCH 0601/1051] arm64: dts: marvell: 7040-db: add comphy references
 to PCIe port

Adding phandle to the PCIe port will allow PCIe driver to configure
appropriate marvell, comphy-cp110 accordingly.

Change-Id: I3f09b271f7a1aae9e04014b8e9e58486f878db3c
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dts b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
index 3c0c889fcaf0..eb64a5eb5535 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
@@ -102,6 +102,8 @@
 
 &cp0_pcie2 {
 	status = "okay";
+	phys = <&cp0_comphy5 2>;
+	phy-names = "pcie-phy0";
 };
 
 &cp0_i2c0 {
-- 
2.17.1

